src/components/data-management/NewProjectModal.jsx

```diff
@@ -55,10 +55,12 @@ const NewProjectModal = (props) => {
 
   return (
     <Modal
+      className='new-project-modal'
```

Comment:
Why not use an id for this instead of class?

Reply:
because they are not supported for modals because you can potentially have more than one